README.md

```diff
@@ -1,3 +1,138 @@
+`TYPE` | `string` | CLTS groups transcription information into three categories: Transcription systems (`ts`), transcription data (`td`) and soundclass systems (`sc`).
+`URITEMPLATE` | `string` | Several CLTS sources provide an online catalog of the graphemes they describe. If this is the case, the URI template specified in this column was used to derive the URL column in graphemes.csv.
+
+
+
+### <a name="table-datafeaturestsv"></a>Table [data/features.tsv](./data/features.tsv)
+
+
+The feature system employed by CLTS describes sounds by assigning values for certain features (constrained by sound type). The permissible values per (feature, sound type) are listed in this table.
+
+property | value
+ --- | ---
+[dc:extent](http://purl.org/dc/terms/extent) | 156
+
+
+
+
+#### Columns
+
+
+Name/Property | Datatype | Description
+ --- | --- | --- 
+[ID](http://cldf.clld.org/v1.0/terms.rdf#id) | `string` | Primary key
+`TYPE` | `string` | 
+`FEATURE` | `string` | 
+`VALUE` | `string` | 
+
+
+
+### <a name="table-datagraphemestsv"></a>Table [data/graphemes.tsv](./data/graphemes.tsv)
+
+
+property | value
+ --- | ---
+[dc:extent](http://purl.org/dc/terms/extent) | 80457
+
+
+
+
+#### Columns
+
+
+Name/Property | Datatype | Description
+ --- | --- | --- 
+`PK` | `integer` | Primary key
+`GRAPHEME` | `string` | Grapheme used in a particular transcription to denote a sound
+`NAME` | `string` | The ordered concatenation of feature values of the denoted sound<br>References [data/sounds.tsv::NAME](#table-datasoundstsv)
+`BIPA` | `string` | The grapheme for the denoted sound in the Broad IPA transcription system
+`DATASET` | `string` | Links to the source of this grapheme<br>References [sources/index.tsv::NAME](#table-sourcesindextsv)
+`FREQUENCY` | `integer` | 
+`URL` | `anyURI` | URL of the grapheme in its source online database
+`IMAGE` | `anyURI` | Image of the typeset grapheme.
+`SOUND` | `anyURI` | Audio recording of the sound being pronounced.
+`EXPLICIT` | `string` | 
+`ALIAS` | `string` | 
+`FEATURES` | `string` | Features of the sound as described in the local feature system of the source dataset
+[NOTE](http://cldf.clld.org/v1.0/terms.rdf#comment) | `string` | 
+
+
+
+### <a name="table-datasoundstsv"></a>Table [data/sounds.tsv](./data/sounds.tsv)
+
+
+property | value
+ --- | ---
+[dc:extent](http://purl.org/dc/terms/extent) | 8647
+
+
+
+
+#### Columns
+
+
+Name/Property | Datatype | Description
+ --- | --- | --- 
+`ID` | `string` | Primary key
+`NAME` | `string` | Ordered list of features + sound type
+`FEATURES` | list of `string` (separated by ` `) | Ordered list of feature values for the sound.<br>References [data/features.tsv::VALUE](#table-datafeaturestsv)
+`GRAPHEME` | `string` | CLTS choses the BIPA grapheme as canonical representative of the graphemes mapped to a sound.
+`UNICODE` | list of `string` (separated by ` / `) | Unicode character names of the codepoints in GRAPHEME
+`GENERATED` | `boolean` | 
```

Comment:
GENERATED indicates whether the CLTS value was *inferred* by our algorithmic procedure (which is active for all diphthongs, all cluster sounds, but also all sounds which we do not label explicitly) or whether no inference was needed, since the sound is explicitly defined.